tests/integration/test_interaction.py

```diff
@@ -122,7 +122,12 @@ def move_item_to_zone(self, item_value, zone_id, action_key):
     def send_input(self, item_value, value):
         element = self._get_item_by_value(item_value)
         self.wait_until_visible(element)
-        element.find_element_by_class_name('input').send_keys(value)
+        # Since virtual-dom may be updating DOM elements by replacing them completely, the
+        # following method must be used to wait for the input box to appear:
+        textbox_visible_selector = '.numerical-input[style*="display: block"] input'
```

Comment:
@mtyaka I don't think so? I was seeing flaky tests and I think one reason is that if we use this pattern (note `wait_until_visible` uses `is_displayed`):
```python
element = find_element_by_class_name('blah')
self.wait_until_visible(element)
```
Then sometimes between the first line and the second line, virtual_dom deletes the element and replaces it with a new one, which then causes this Selenium error: "The element is no longer attached to the DOM."